<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScmpRescaleWindow</class>
 <widget class="QMainWindow" name="ScmpRescaleWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>602</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SCMP Rescale / Import Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="sourceMapGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Select .scmap file to resize / merge:</string>
    </property>
    <widget class="QPushButton" name="sourceMapButton">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sourceMapLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Path to .scmp file to rescale and insert into target</string>
     </property>
    </widget>
    <widget class="QLabel" name="sourceSizeLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>source size</string>
     </property>
    </widget>
    <widget class="QLabel" name="sourceVersionLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>source version</string>
     </property>
    </widget>
    <widget class="QLabel" name="sourceLayerCountLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>source layers</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="sourceResizeGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Resize to:</string>
    </property>
    <widget class="QLabel" name="sourceNewWidthLabel">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Resize to width (px)</string>
     </property>
    </widget>
    <widget class="QLabel" name="sourceNewHeightLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Resize to Height (px)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="sourceNewWidthComboBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="sourceNewHeightComboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="targetMapGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>441</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Select new .scmap file / existing .scmap to merge into</string>
    </property>
    <widget class="QPushButton" name="targetMapButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="targetMapLineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Path to .scmp file into which to insert the source map.  Created if not exist</string>
     </property>
    </widget>
    <widget class="QLabel" name="targetVersionLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>target version</string>
     </property>
    </widget>
    <widget class="QLabel" name="targetLayerCountLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>target layers</string>
     </property>
    </widget>
    <widget class="QLabel" name="targetSizeLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>target size</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="saveOptionsGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>441</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Save Options</string>
    </property>
    <widget class="QRadioButton" name="overWriteModeRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Overwrite / Create</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="mergeModeRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Merge Into Existing</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="mergePositionFrame">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>261</width>
       <height>201</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Merge at horizontal position (px)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>178</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Merge at vertical position (px)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="sourceHorizontalPositionSpinBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="sourceHorizontalRightPositionSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="sourceVerticalPositionSpinBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="sourceVerticalTopPositionSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="sourceHorizontalLeftPositionSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="sourceVerticalBottomPositionSlider">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QCheckBox" name="additiveMergeCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Merge heightmap additively</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>go!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sourceMapLineEdit</tabstop>
  <tabstop>targetMapLineEdit</tabstop>
  <tabstop>sourceHorizontalPositionSpinBox</tabstop>
  <tabstop>sourceVerticalPositionSpinBox</tabstop>
  <tabstop>goButton</tabstop>
  <tabstop>sourceMapButton</tabstop>
  <tabstop>targetMapButton</tabstop>
  <tabstop>overWriteModeRadioButton</tabstop>
  <tabstop>mergeModeRadioButton</tabstop>
  <tabstop>sourceHorizontalLeftPositionSlider</tabstop>
  <tabstop>sourceHorizontalRightPositionSlider</tabstop>
  <tabstop>sourceVerticalBottomPositionSlider</tabstop>
  <tabstop>sourceVerticalTopPositionSlider</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
